openedx/edx-ora2
# [BB-6151] Update TinyMCE Editor static file references to upgraded version 5.x

**TL;DR -** Replaces the version 4.x references of TinyMCE static files with version 5.x and removes the workaround required for TMCE 4 from oa_editor_tinymce.js

JIRA: - NA

**What changed?**

The edx-platform's TinyMCE is being upgraded from version 4.x to 5.x (v5.5.1) at https://github.com/openedx/edx-platform/pull/30335. The upgrade breaks the ORA WYSIWYG editor as the TinyMCE files referenced in the [oa_editor_tinymce.js](https://github.com/openedx/edx-ora2/blob/master/openassessment/xblock/static/js/src/lms/editors/oa_editor_tinymce.js#L7) are hard-coded to v4.x file paths.

- In this PR these paths are update to reference the v5.x files
- Corresponding changes are made in the TinyMCE init function
- Removes a TinyMCE 4.x specific workaround
- Updates all the static files using `make static`


**Developer Checklist**


**Testing Instructions**

1. Checkout to the PR branch of https://github.com/openedx/edx-platform/pull/30335
2. Add a ORA block in a course and set the editor to WYSIWYG
3. Open the Unit in the LMS and notice that the Editor doesn't load for input
4. Install the `open-craft:tecoholic/BB-6151-upgrade-tinymce-to-v5` branch of the ORA in to lms
5. Restart the LMS
6. Revisit the test unit in LMS and verify that the Editor is functioning as expected

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @openedx/content-aurora
